includes/render/class-linked-image-reusable-block.php

```diff
@@ -50,7 +50,7 @@ protected function block_content(): string {
         $dashed_code_name =  str_replace('_', '-', $this->code_name);
 		return "
 <!-- wp:image -->
-	<figure class=\"wp-block-image figure_$class_name fixed-size-linked-img\"><a href=\"$escaped_link\"><img src=\"$escaped_image\" alt=\"link to $escaped_readable_name\"/></a></figure>
+	<figure class=\"wp-block-image figure_$class_name fixed-size-linked-img\"><a href=\"$escaped_link\"><img style=\"width: 32px; height: 32px;\" src=\"$escaped_image\" alt=\"link to $escaped_readable_name\"/></a></figure>
```

Comment:
If the style is added here, do we still need to call that `register_block_style`?

Reply:
I can test it without the call. I'm not entirely sure why the styling wasn't added directly here at the first place, so I thought it could be there just in case.

Reply:
Working without registering the style. Removing it.